### DISCUSSION TITLE
automation documentation

### DISCUSSION THREAD

**Discussion by craigsmooth** [Security-Onion-Solutions/securityonion#7907] [Unsupported Versions] [answered]
Is there any documentation for the "automation" flag that can be passed to "so-setup". I know there are a lot of examples for the different roles but is there a reference for all possible variables and their descriptions/purpose somewhere? Thank you.

**Comment by dougburks** [comment, marked as answer]
The automation flag is for our automated testing and is not supported for any other use.

**Reply by craigsmooth** [reply]
Ahhh, sorry @dougburks . I guess what I am asking is.. is there any documentation for the file format of the "silent install" files located here:
https://github.com/Security-Onion-Solutions/securityonion/tree/master/setup/automation
Thanks!!

**Reply by dougburks** [reply]
As I mentioned in my previous reply, those files are for our automated testing only. They are undocumented and unsupported.

**Comment by craigsmooth** [comment]
Ah. Gotcha. Thanks Doug. I thought this might be the replacement
functionality for the old unattended setup answer file.

On Mon, May 9, 2022 at 5:51 AM Doug Burks ***@***.***> wrote:

> As I mentioned in my previous reply, those files are for our automated
> testing only. They are undocumented and unsupported.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/7907#discussioncomment-2713664>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABNM3RG4G4IZIKNLN42LJZLVJDU23ANCNFSM5VGAKXSQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <Security-Onion-Solutions/securityonion/repo-discussions/7907/comments/2713664
> @github.com>
>
-- 
Sent from Gmail Mobile
